src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="Styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="Styles/style.css">
+  </head>
+  <body>
+    <div data-qa="card"class="card">
+      <img
+        class="card__photo"
+        src="./images/imac.jpeg"
+        alt="Retina 5K Monoblock"
+      >
+      <h1 class="card__title"> APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__reviews">
+        <div class="stars stars--0">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__container-review">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <h2 class="card__price-word">Price:</h2>
+        <p class="card__digital-price">$2,199</p>
+      </div>
+
+      <a
+        data-qa="hover"
+        class="card__button-link"
+        href="/"
+      >
+        BUY
+      </a>
+
+    </div>
```

Comment:
```suggestion
      </a>
    </div>
```
Don't leave new lines between parent and child elements